sphinx_gallery/gen_rst.py

```diff
@@ -714,8 +731,13 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
                      mode='w', encoding='utf-8') as f:
         if time_elapsed >= gallery_conf["min_reported_time"]:
             example_rst += ("**Total running time of the script:**"
-                            " ({0: .0f} minutes {1: .3f} seconds)\n\n".format(
-                                time_m, time_s))
+                            " ({0: .0f} minutes {1: .3f} seconds)\n\n"
+                            .format(time_m, time_s))
+        if show_memory:
+            peak_mem = max(memory_measurements)
+            example_rst += ("**Peak memory usage:** {0: .3f}MB\n\n"
```

Comment:
I think `.1f` (or even `d`) is sufficient here

Reply:
... and there should be a space between the number and `MB`

Reply:
I'll take care when I am back at my desk ;)